app/static/src/app/types/payloadTypes.ts

```diff
@@ -4,6 +4,8 @@ export interface SetAppPayload {
     appName: string
     baseUrl: string
     appsPath: string
+    internationalisation: boolean
+    defaultLanguage: "en" | "fr" | "pt"
```

Comment:
Here's the second place this list of 3 possibilities appears - this will complicate adding new languages. I suggest that perhaps the backend one could be `string`